.github/workflows/nuget.yml

```diff
@@ -0,0 +1,35 @@
+name: Nuget
+
+on:
+  create:
+    tags:
```

Comment:
Maybe nuget package only shall be created on release? Not tag?

Reply:
You are right, I changed it to when ever the release is published.